lib/editor-settings.php

```diff
@@ -77,6 +78,7 @@ function gutenberg_get_common_block_editor_settings() {
  */
 function gutenberg_extend_post_editor_settings( $settings ) {
 	$settings['__unstableEnableFullSiteEditingBlocks'] = gutenberg_is_fse_theme();
+	$settings['__unstableEnablePostBlocks']            = gutenberg_is_experiment_enabled( 'gutenberg-post-blocks' );
```

Comment:
Should we add `|| gutenberg_is_fse_theme()`.  It doesn't make sense for me to have an FSE theme without these blocks.

Reply:
We do add them if `gutenberg_is_fse_theme`... The only difference in the current status of this PR is `template part`, which is enabled only in FSE.